### ISSUE TITLE
SS 3.3 Viewer groups not visible

### ISSUE THREAD

**Issue by MilesSummers** [silverstripe/silverstripe-secureassets#45]
Upgraded to SS 3.3.x and the Viewer groups dropdown is not showing it's entries (they exist in html)
Oddly enough if I collapse my window so the Save/Delete bar is hidden (the dropdown has to pop upwards) they display.

![screenshot_20160601_162928](https://cloud.githubusercontent.com/assets/4961437/15700330/9ebf32a8-2816-11e6-91f2-346a9d516b11.png)

**Comment by tractorcow** [comment]
Should be easy to fix; Will have to have a look at asset admin to see if there's a CSS rule we can tweak.

**Comment by dhensby** [comment]
@hafriedlander is this related to the upgrade of chzn?

**Comment by dhensby** [comment]
possibly related to https://github.com/silverstripe/silverstripe-framework/pull/5309 and the fact we dropped "rise-up" customisation

**Comment by colintucker** [comment]
Hey all,

@dhensby @hafriedlander I believe I've found the cause (after much frustration and head scratching... :P).  It works when you edit the folder using the `GridField` under `AssetAdmin` (although the JavaScript and CSS is borked).  You'll notice in the source that the `TreeDropdownField` has it's `data-url-tree` set to:

```
data-url-tree="admin/assets/EditForm/field/File/item/1/ItemEditForm/field/ViewerGroups/tree"
```

However, when you try to use the field when it's been embedded in the `AssetAdmin` EditForm (again, JavaScript and CSS is borked for the Secure Assets fields), you'll see this because the fields from the folder are pushed to the EditForm:

```
data-url-tree="admin/assets/EditForm/field/ViewerGroups/tree"
```

So when the `TreeDropdownField` is clicked, and it tries to load the tree via Ajax, it can't find it! There is no identifiable information for it to locate the field object.

Additionally, because the framework `TabSet.js` defines a global `ajaxComplete`, when the Ajax load for the ViewerGroups field completes, it triggers a `click` on the DetailsView tab, which results in the dropdown closing, and this:

![Navigation warning dialog](http://i.imgur.com/e9XjEhF.png)

Yuck!  ☹️ 

I just added a workaround for these problems to my [silverstripe-moderno-admin](https://github.com/praxisnetau/silverstripe-moderno-admin) module. Bit of a band-aid solution though!

I hope this can shed some light on what's going on.

**Comment by robbieaverill** [comment]
Looks like #55 and this are the same

**Comment by dhensby** [comment]
@colintucker / @robbieaverill I don't think this is related to #55. 

The issue reported here is that the HTML is all fine but the dropdown is obscured.

@colintucker I believe what you're reporting may well be the same as #55, but I'm unable to replicate it.

**Comment by dhensby** [comment]
FYI - I can't replicate this problem

**Comment by robbieaverill** [comment]
Could we please re-evaluate this problem on 1.1.5, which has the fix for #55?

**Comment by robbieaverill** [comment]
Feel free to reopen if this is still an issue. Closing since it hasn't received any feedback in six months or so since @dhensby said he couldn't reproduce the issue.
